api/apps/api/src/modules/projects/project.api.entity.ts

```diff
@@ -131,6 +136,13 @@ export class Project extends TimeUserEntityMetadata {
   @OneToOne(() => ProjectBlm)
   projectBlm!: ProjectBlm;
 
+  @ApiProperty()
+  @Column('enum', {
+    name: 'sources',
+    enum: ProjectSourcesEnum,
+  })
+  sources?: string;
```

Comment:
I think we should mark `sources` attribute as mandatory since it can't hold null values. Also we should use the enum as type right?

```suggestion
  sources!: ProjectSourcesEnum;
```